src/components/Button.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+
+const Button = props => (
+  <button type="button">
+    {props.name}
+  </button>
+);
+
```

Comment:
Let's add prop-types for the props, that was the mean reason you added it in milestone 1.

Once you have done this, I don't think you need to disable the rule at line 1 in this file.